core/src/main/java/io/cucumber/core/plugin/ProtobufFormatterAdapter.java

```diff
@@ -0,0 +1,28 @@
+package io.cucumber.core.plugin;
+
+import io.cucumber.core.exception.CucumberException;
+import io.cucumber.plugin.ConcurrentEventListener;
+import io.cucumber.plugin.EventListener;
+import io.cucumber.plugin.event.EventPublisher;
+
+import java.io.OutputStream;
+import java.lang.reflect.InvocationTargetException;
+
+public class ProtobufFormatterAdapter implements ConcurrentEventListener {
+    public static final String PROTOBUF_FORMATTER = "io.cucumber.core.gherkin8.formatter.ProtobufFormatter";
+    private final EventListener delegate;
+
+    public ProtobufFormatterAdapter(OutputStream out) {
+        try {
+            Class<EventListener> delegateClass = (Class<EventListener>) getClass().getClassLoader().loadClass(PROTOBUF_FORMATTER);
```

Comment:
For now this is okay. 

But we should upgrade the plugin system to make registering plugins via SPI possible. Then the protobuf formatter could live in its own module.

Reply:
Completely agree - I added it to the gherkin8 module out of pure laziness since I think we're still in spike mode. Let's do that before we merge this.

Reply:
Cool!